create-app.js

```diff
@@ -0,0 +1,203 @@
+    '@openzeppelin/contracts',
+    'chai',
+    'ethereum-waffle',
+    'ethers',
+    'react',
+    'react-dom',
+    'next',
+  ];
+  /**
+   * Default devDependencies.
+   */
+  const devDependencies = ['eslint', 'eslint-config-next', 'hardhat'];
+  /**
+   * TypeScript projects will have type definitions and other devDependencies.
+   */
+  if (typescript) {
+    devDependencies.push('typescript', '@types/react', '@types/node');
+  }
+  /**
+   * Install package.json dependencies if they exist.
+   */
+  if (dependencies.length) {
+    console.log();
+    console.log('Installing dependencies:');
+    for (const dependency of dependencies) {
+      console.log(`- ${chalk.cyan(dependency)}`);
+    }
+    console.log();
+
+    await installDep.install(root, dependencies, installFlags);
+  }
+  /**
+   * Install package.json devDependencies if they exist.
+   */
+  if (devDependencies.length) {
+    console.log();
+    console.log('Installing devDependencies:');
+    for (const devDependency of devDependencies) {
+      console.log(`- ${chalk.cyan(devDependency)}`);
+    }
+    console.log();
+
+    const devInstallFlags = { devDependencies: true, ...installFlags };
+    await installDep.install(root, devDependencies, devInstallFlags);
+  }
+  console.log();
+  /**
+   * Copy the template files to the target directory.
+   */
+  await cpy('**', root, {
+    parents: true,
+    // cwd: path.join(__dirname, 'templates', template), //for seperate templated folders
+    cwd: path.join(__dirname, 'template'), //for single template
+    rename: (name) => {
+      switch (name) {
+        case 'gitignore':
+        case 'eslintrc.json': {
+          return '.'.concat(name);
+        }
+        // README.md is ignored by webpack-asset-relocator-loader used by ncc:
+        // https://github.com/vercel/webpack-asset-relocator-loader/blob/e9308683d47ff507253e37c9bcbb99474603192b/src/asset-relocator.js#L227
+        case 'README-template.md': {
+          return 'README.md';
+        }
+        default: {
+          return name;
+        }
+      }
+    },
+  });
+
+  if (gitInit.tryGitInit(root)) {
+    console.log('Initialized a git repository.');
+    console.log();
+  }
+
+  let cdpath = '';
+  if (path.join(originalDirectory, appName) === appPath) {
+    cdpath = appName;
+  } else {
+    cdpath = appPath;
+  }
+
+  console.log(`${chalk.green('Success!')} Created ${appName} at ${appPath}`);
+  console.log('Inside that directory, you can run several commands:');
+  console.log();
+  console.log(chalk.cyan(`  ${displayedCommand} ${useYarn ? '' : 'run '}dev`));
+  console.log('    Starts the development server.');
+  console.log();
+  console.log(
+    chalk.cyan(`  ${displayedCommand} ${useYarn ? '' : 'run '}build`)
+  );
+  console.log('    Builds the app for production.');
+  console.log();
+  console.log(chalk.cyan(`  ${displayedCommand} start`));
+  console.log('    Runs the built app in production mode.');
+  console.log();
```

Comment:
```suggestion
```

Let's remove these since we have the actual instructions below them. We will add the instructions for building and deploying later!